lib/ledger-node-operations.js

```diff
@@ -43,15 +71,35 @@ module.exports = class LedgerNodeOperations {
    *
    * operation - the operation to validate.
    * callback(err) - the callback to call when finished.
-   *   err - An Error if an error occurred, null otherwise.
+   *   err - A BedrockError if an error occurred, null otherwise.
    */
   validate(operation, callback) {
+    if(!(operation && typeof operation === 'object')) {
+      throw new BedrockError('Operation must be an object.', 'TypeError');
```

Comment:
Should be `return callback(new BedrockError(...))`.

Reply:
Again, we use `TypeError` for this sort of thing, not `BedrockError` -- if we're going to be consistent.